lib/game/player_type/easy_computer_player.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Game.PlayerType.EasyComputerPlayer do
+  @moduledoc """
+  The module that handles an Easy Computer Players move
+  """
+
+  alias Game.PlayerType.PlayerBehaviour
+  @behaviour PlayerBehaviour
+
+  @impl PlayerBehaviour
+  def valid_input(board, communicator) do
+    valid_move = board
+    |> Map.filter(fn {_key, val} -> val == :empty end)
```

Comment:
I see a couple places in your code where you check whether a space is `:empty` in different ways. Maybe you could create a `Board` helper function (i.e. `space_empty?`).

Also, in this specific instance maybe you would want a `Board` helper function to return the `available_spaces`? I think that logic belongs there rather than here.